domain-specific-datasets/app/pipeline.yaml

```diff
@@ -0,0 +1,471 @@
+distilabel:
```

Comment:
I think pushing the yaml approach is cool but more useful for more advanced users.

Reply:
We use the yaml to share and execute the pipeline.